docs/source/data_conversion/data_conversion.rst

```diff
@@ -0,0 +1,278 @@
+.. include:: ../substitutions.rst
+
+Data Conversion in GraphNeT\ |graphnet-header|
+==============================================
+
+GraphNeT comes with powerful data conversion code that can convert data formats specific to experiments to deep learning friendly data formats.
+
+Data conversion in GraphNeT follows a reader/writer scheme, where the :code:`DataConverter` does most of the heavy lifting.
+
+.. image:: ../../../assets/images/dataconverter.svg
+    :width: 500
+    :alt: Illustration of the reader/writer scheme of data conversion in GraphNeT.
+    :align: right
+    :class: with-shadow
+    
+    
+    
+
+In the illustration, the "reader" module represents an experiment-specific implementation of :code:`GraphNeTFileReader`, able to parse your data files. The "Writer" module denotes a :code:`GraphNeTWriter` module that saves the interim data format from :code:`DataConverter` to disk. 
```

Comment:
maybe specify that the interim data formats currently supported are parquet and sqlite.